tests/pnpm-sync-api-test/src/test/pnpmSyncCopy.test.ts

```diff
@@ -88,6 +100,14 @@ describe('pnpm-sync-api copy test', () => {
     // after copy action, the destination folder should exists
     expect(fs.existsSync(destinationPath)).toBe(true);
 
+    // and temp folder and file should be deleted
+    expect(fs.existsSync(testTempFolder)).toBe(false);
+    expect(fs.existsSync(testTempFile)).toBe(false);
+
+    // and the real files should be there
+    expect(fs.existsSync(path.join(destinationPath, 'src/index.ts'))).toBe(true);
+    expect(fs.existsSync(path.join(destinationPath, 'dist/index.js'))).toBe(true);
+
```

Comment:
It might be useful to have a second Jest test that:

1. Copies a more complex directory tree (for better code coverage of your algorithm)
2. Incrementally copies from a different directory tree, to show how items get added/removed
3. Intercepts file operations, so that it can count the number of times that the algorithm called interesting API's.  (For example, if our test is syncing 100 files with only 10 differences, then it would indicate a performance oversight if Jest reports 500 calls to an API like `exists()`.)  Although the ordering of API calls may be nondeterministic, the count of API calls probably is deterministic and could be tracked easily using a Jest snapshot.

Reply:
Thanks for the suggestion. 
I added a separated test for incremental copy